extension/src/experiments/webview/contract.ts

```diff
@@ -93,7 +93,7 @@ export interface Column extends ColumnAggregateData {
 
 export type TableData = {
   changes: string[]
-  cliError: string | undefined
```

Comment:
If a key in the object sent is undefined, the key will not exist in the data received by the webview. I believe this is because VSCode expects a json serializable object. 

```
         * Post a message to the webview content.
         *
         * Messages are only delivered if the webview is live (either visible or in the
         * background with `retainContextWhenHidden`).
         *
         * @param message Body of the message. This must be a string or other json serializable object.
```